### PR TITLE
Post logic

### PR DESCRIPTION
**Pull request by JowelTisso**
* This PR is for post features :
* Create, edit, delete post

### CODE REVIEW

File: src/store/postSlice.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  "post/editPost",
+  async ({ postId, postData }, { rejectWithValue }) => {
+    try {
+      const res = await POST(`${API.EDIT_POST}/${postId}`, { postData });
+      if (res?.status === 200 || res?.status === 201) {
+        return res?.data.posts;
+      }
+    } catch (err) {
+      console.log(err);
+      rejectWithValue(err.message);
```
Comment (sanjitsarkar) [inline src/store/postSlice.js:49]: I like how you create separate function for error handling and Objects for API endpoints

**Review by sanjitsarkar — APPROVED** [review]
- Code is well structured and modulerized
Good to merge.

File: src/utils/axiosHelper.js [hunk 2]
```diff
@@ -54,9 +54,13 @@ export const POST_AUTH = async (url, body) => {
  */
 export const DELETE = async (url, body) => {
   try {
-    return await axios.delete(url, body, {
-      headers: { authorization: getUserToken() },
-    });
+    return await axios.delete(
+      url,
+      {
+        headers: { authorization: getUserToken() },
```
Comment (sanjitsarkar) [inline src/utils/axiosHelper.js:60]: you can use axios interceptor method for passing authorization token

**Review by sanjitsarkar — APPROVED** [review]

**Review by SukanyaSen530 — APPROVED** [review]
Clean and structured code, easy to understand.

**Pull request merged**